scripts/update-patches.sh

```diff
@@ -13,13 +13,13 @@ GLUONDIR="$(pwd)"
 for module in $GLUON_MODULES; do
 	echo "--- Updating patches for module '$module' ---"
 
-	rm -rf "${GLUON_PATCHESDIR}/$module"
+	rm -rf "${GLUON_PATCHESDIR:?}/$module"
 
 	cd "$GLUONDIR"/"$module"
 
 	n=0
 	for commit in $(git rev-list --reverse --no-merges base..patched); do
-		let n=n+1
+		(( n++ ))
```

Comment:
Which part doesn't shellcheck like, the `let` or the `n=n+1`? I prefer `let n++` to `(( n++ ))`.

Reply:
related to the let. See https://github.com/koalaman/shellcheck/wiki/SC2219

Reply:
Change `n++` to `++n`, otherwise this fails the `set -e` check when *n* is 0.